.github/workflows/deploy-feature-envs.yaml

```diff
@@ -54,12 +59,14 @@ jobs:
 
       - name: Build docker image with PR aleph-node binary
         env:
-          IMAGE_TAG: ${{ env.FE_IMAGETAG_PREFIX }}${{ steps.get_branch.outputs.branch_imagetag_full }}
+          IMAGE_TAG:
```

Comment:
just checking - can you have that value in a new line without any `\`, `|` or `>`?

Reply:
Yes - please refer to https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines , "Flow" scalar styles ( , ", ') section